public/app/features/auth-config/fields.tsx

```diff
@@ -890,6 +919,22 @@ export function fieldMap(provider: string): Record<string, FieldData> {
         message: t('auth-config.fields.domain-hint-valid-domain', 'This field must be a valid domain.'),
       },
     },
+    loginPrompt: {
+      label: t('auth-config.fields.login-prompt-label', 'Login prompt'),
+      type: 'select',
+      description: t(
+        'auth-config.fields.login-prompt-description',
+        'Indicates the type of user interaction when the user logs in with the IdP.'
+      ),
+      multi: false,
+      options: [
+        { value: '', label: '' },
+        { value: 'login', label: t('auth-config.fields.login-prompt-login', 'Login') },
```

Comment:
I think we don't need the translations for these, wdyt?

Reply:
A `/* eslint-disable-next-line @grafana/i18n/no-untranslated-strings */` comment can be used to suppress this.

But is this a label to show in the UI? I see the value is left untranslated, but the label property for dropdown options should probably be translated.